src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -53,80 +55,116 @@
 			"\tsend \"You're being followed by an %entity%!\" to target of entity"})
 @Since("<i>unknown</i> (before 2.1)")
 public class ExprTarget extends PropertyExpression<LivingEntity, Entity> {
+
 	static {
 		Skript.registerExpression(ExprTarget.class, Entity.class, ExpressionType.PROPERTY,
 				"[the] target[[ed] %-*entitydata%] [of %livingentities%]",
 				"%livingentities%'[s] target[[ed] %-*entitydata%]");
 	}
-	
+
 	@Nullable
 	EntityData<?> type;
-	
-	@SuppressWarnings({"unchecked", "null"})
+
+	@SuppressWarnings("unchecked")
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parser) {
 		type = exprs[matchedPattern] == null ? null : (EntityData<?>) exprs[matchedPattern].getSingle(null);
 		setExpr((Expression<? extends LivingEntity>) exprs[1 - matchedPattern]);
 		return true;
 	}
-	
+
 	@Override
-	protected Entity[] get(final Event e, final LivingEntity[] source) {
+	protected Entity[] get(Event e, LivingEntity[] source) {
 		return get(source, new Converter<LivingEntity, Entity>() {
 			@Override
 			@Nullable
-			public Entity convert(final LivingEntity en) {
+			public Entity convert(LivingEntity en) {
 				if (getTime() >= 0 && e instanceof EntityTargetEvent && en.equals(((EntityTargetEvent) e).getEntity()) && !Delay.isDelayed(e)) {
-					final Entity t = ((EntityTargetEvent) e).getTarget();
-					if (t == null || type != null && !type.isInstance(t))
+					Entity target = ((EntityTargetEvent) e).getTarget();
+					if (target == null || type != null && !type.isInstance(target))
 						return null;
-					return t;
+					return target;
 				}
-				return Utils.getTarget(en, type);
+				return getTarget(en, type);
 			}
 		});
 	}
-	
+
 	@Override
 	public Class<? extends Entity> getReturnType() {
 		return type != null ? type.getType() : Entity.class;
 	}
-	
+
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
+	public String toString(@Nullable Event e, boolean debug) {
 		if (e == null)
 			return "the target" + (type == null ? "" : "ed " + type) + (getExpr().isDefault() ? "" : " of " + getExpr().toString(e, debug));
 		return Classes.getDebugMessage(getAll(e));
 	}
-	
+
 	@Override
-	public boolean setTime(final int time) {
+	public boolean setTime(int time) {
 		return super.setTime(time, EntityTargetEvent.class, getExpr());
 	}
-	
+
 	@Override
 	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
+	public Class<?>[] acceptChange(ChangeMode mode) {
 		if (mode == ChangeMode.SET || mode == ChangeMode.DELETE)
 			return CollectionUtils.array(LivingEntity.class);
 		return super.acceptChange(mode);
```

Comment:
We can just return null here

Reply:
This is existing behavior. The super.acceptChange already returns null.

Reply:
`super.acceptChange` won't just return null, and shouldn't. It refers to the default entityChanger, e.g. for `add speed to target of {_e}`